src/mem/ruby/structures/CacheMemory.cc

```diff
@@ -73,6 +73,8 @@ CacheMemory::CacheMemory(const Params &p)
               p.start_index_bit, p.ruby_system),
     tagArray(p.tagArrayBanks, p.tagAccessLatency,
              p.start_index_bit, p.ruby_system),
+    atomicALUArray(p.atomicALUs, p.atomicLatency,
```

Comment:
Here you should be able to use a function that's something like `cyclesToTicks(p.atomicLatency)`